sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -84,46 +92,85 @@ async function main() {
     throw appComponentCreationResult.body.error;
   }
 
-  // Creating the test run
-  const testRunCreationResult = await client.path("/testruns/{testRunId}", testRunId).patch({
+  // Creating/Updating the test run
+  const testRunPoller = await beginCreateOrUpdateTestRun(client, testRunId, {
```

Comment:
should we move this to its own sample to simplify things or do we need to perform all these operations sequentially in every scenario?

Reply:
The scenario is sequential where user will create test and upload file, then testrun can be created.